org.eclipse.jdt.ls.core/src/org/eclipse/jdt/ls/core/internal/managers/GradleProjectImporter.java

```diff
@@ -164,7 +168,13 @@ public void importToWorkspace(IProgressMonitor monitor) throws CoreException {
 		subMonitor.setTaskName(IMPORTING_GRADLE_PROJECTS);
 		JavaLanguageServerPlugin.logInfo(IMPORTING_GRADLE_PROJECTS);
 		subMonitor.worked(1);
-		directories.forEach(d -> importDir(d, subMonitor.newChild(1)));
+		MultiStatus failedStatusCollection = new MultiStatus(IConstants.PLUGIN_ID, IStatus.WARNING, "Errors happen when importing Gradle projects", null);
```

Comment:
I would change the error message to `Errors occurred during import of Gradle projects`.